### ISSUE TITLE
support vs 2019 please

### ISSUE THREAD

**Issue by JaWiD2** [jgyo/remarker#7]
❤🙏🏻

**Comment by kirbatious** [comment]
Given @jgyo has abandoned this project, we need to send all of our heart emoji towards @negrutiu , who has forked the project here and fixed it for VS2017 https://github.com/negrutiu/remarker/releases/tag/v2.2

Any chance of answering our prayers @negrutiu ?

**Comment by negrutiu** [comment]
Hey guys. You may want to check out my [latest release](https://github.com/negrutiu/remarker/releases). Happy coding!

**Comment by JaWiD2** [comment]
Hi @negrutiu , Recently updated my Visual Studio 2019 to version 16.1.5 and your last released (v2.3) no longer work!
Can you fix it please?
Thanks for your kind ❤

**Comment by negrutiu** [comment]
Problem fixed in my [latest release](https://github.com/negrutiu/remarker/releases).
Thanks for reporting!

**Comment by kirbatious** [comment]
Hi @negrutiu - do you know if there's any way to get your fork of this onto the Visual Studio Marketplace? My company doesn't want us sideloading vsix files :(

**Comment by tgaldi** [comment]
@negrutiu Will you be updating this to VS 2022?

**Comment by jgyo** [comment]
No. I am retired and don't have the resources now to support upgrades.

**Comment by kirbatious** [comment]
Thanks for responding Gil @jgyo, and I hope you're enjoying your retirement. 

Is there any way to transfer ownership of this repo and the associated extension access on VS marketplace to someone like @negrutiu who is able to keep it going?

**Comment by jgyo** [comment]
Anyone that wants to update it can clone it from GitHub and make any
changes that they wish. The license for it allows for that.

**Comment by kirbatious** [comment]
I do know how GitHub works (eg @negrutiu forked a version for vs2019) - it's the VS extension marketplace https://marketplace.visualstudio.com/items?itemName=GilYoder.Remarker-18580 that I would love to get updated so I don't have to sideload it (which my work are not keen on). I guess anyone can set up a Remarker 2.0 page?

**Comment by jgyo** [comment]
>> I guess anyone can set up a Remarker 2.0 page?

Yep. I've done that once myself for a WPF utility that was first written by
another programmer.
